pkg/exchange/okex/exchange.go

```diff
@@ -170,6 +170,8 @@ func (e *Exchange) QueryMarkets(ctx context.Context) (types.MarketMap, error) {
 			// OKEx does not offer minimal notional, use 1 USD here.
 			MinNotional: fixedpoint.One,
 			MinAmount:   fixedpoint.One,
+
+			ContractValue: fixedpoint.MustNewFromString(instrument.ContractValue),
```

Comment:
you can update the type instrument.ContractValue , so we can parse it directly from JSON